src/EditorFeatures/Core/Implementation/InlineRename/CommandHandlers/RenameCommandHandler_RenameHandler.cs

```diff
@@ -80,6 +73,25 @@ private void ExecuteRenameWorker(RenameCommandArgs args, CancellationToken cance
                 return;
```

Comment:
My guess is that this happens in some projection scenario where the caret doesn't map into our subjectbuffer...